docs/README.md

```diff
@@ -38,7 +38,7 @@ installed. Also install the following libraries:
 ```sh
 $ sudo gem install jekyll jekyll-redirect-from rouge
 # Following is needed only for generating API docs
-$ sudo pip install sphinx pypandoc mkdocs
+$ sudo pip install sphinx mkdocs numpy
```

Comment:
Without `numpy` I'm getting these errors when I try to build the PySpark ML API docs:

```
updating environment: 0 added, 2 changed, 0 removed
reading sources... [100%] pyspark.mllib                      
WARNING: autodoc: failed to import module 'ml' from module 'pyspark'; the following exception was raised:
No module named 'numpy'
WARNING: autodoc: failed to import module 'ml.param' from module 'pyspark'; the following exception was raised:
No module named 'numpy'
...
```

Reply:
@nchammas . For `numpy` addition, we had better go with another PR separately. It will be a minor. Please just make a PR for that.

Reply:
Split to #27390.